assets/js/pages/search.js

```diff
@@ -2,7 +2,7 @@
   window.reloadPackages = function(currentFunctionPage, currentPackagePage){
     $('html, body').animate({ scrollTop: 0 }, 'slow');
   	$.ajax({
-  		url: "/search_packages?q="+urlParam('q') + "&page=" + currentPackagePage,
+  		url: "/search_packages?q="+urlParam('q') + "&page=" + currentPackagePage +"&viewer_pane="+urlParam('viewer_pane'),
```

Comment:
Yes, just found out this is actually a really bad fix. The real problem is that it binds the rstudio_layout to it due to the header (causing the historynavigation to dissapear) . I'll checkout the original file. Can I add a specific header for ajax request that only change a part of the file? because I already had to adapt the rstudio_redirect for the taskviews (replaced only the 'view' pane, but it seems cleaner to add a general method for all such ajax-requests.